src/domain/safe/safe.repository.interface.ts

```diff
@@ -23,10 +23,7 @@ export interface ISafeRepository {
     offset?: number;
   }): Promise<Page<Transfer>>;
 
-  clearCollectibleTransfers(args: {
-    chainId: string;
-    safeAddress: string;
-  }): Promise<void>;
+  clearTransfers(args: { chainId: string; safeAddress: string }): Promise<void>;
```

Comment:
Just renaming here. This function calls `TransactionApi.clearTransfers` which clears all the `transfers`, not only _collectibles_ (ERC-721).